h2o-py/h2o/frame.py

```diff
@@ -735,39 +748,85 @@ def digamma(self):
     def trigamma(self):
         return self._unop("trigamma")
 
+
     @staticmethod
     def mktime(year=1970, month=0, day=0, hour=0, minute=0, second=0, msec=0):
-        """All units are zero-based (including months and days).
-        Missing year is 1970.
-
-        Parameters
-        ----------
-          year : int, H2OFrame
-            the year
-
-          month: int, H2OFrame
-            the month
-
-          day : int, H2OFrame
-            the day
-
-          hour : int, H2OFrame
-            the hour
+        """Deprecated, use `.moment()` instead."""
```

Comment:
Fair enough, I've expanded the help for this function and moved it into the "deprecated" "section".